api/taxonomies/views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import generics
+
+from api.taxonomies.serializers import TaxonomySerializer
+from api.base.views import JSONAPIBaseView
+
+import json
+
+class PlosTaxonomyMixin(JSONAPIBaseView, generics.RetrieveAPIView):
+    permission_classes = (
```

Comment:
For now, add `drf_permissions.IsAuthenticatedOrReadOnly` to the permission_classes! Later on we can see if we need others, but this should do for now just to have something.